### PR TITLE
Spinquic job for LoLa

### PR DESCRIPTION
**Pull request by maolson-msft**

### CODE REVIEW

**Review by nibanks — APPROVED** [review]

File: .azure/azure-pipelines.ci.yml [hunk 1]
```diff
@@ -673,6 +673,15 @@ stages:
       platform: windows
       tls: schannel
       allocFail: 100
+  - template: ./templates/run-spinquic.yml
+    parameters:
+      image: windows-2022
+      platform: windows
+      tls: schannel
+      allocFail: 100
```
Comment (anrossi) [inline .azure/azure-pipelines.ci.yml:681]: does this use the same `CXPLAT_ALLOC` routines, or have its own allocator?

Comment (maolson-msft) [reply .azure/azure-pipelines.ci.yml:681]: Same ones. LoLa uses a different "datapath" but same "platform" as winuser.

**Review by csujedihy — APPROVED** [review]
It would be nice if you can put down some notes for the bugfix in the PR description.

**Review by nibanks — APPROVED** [review]

**Comment by maolson-msft** [comment]
> It would be nice if you can put down some notes for the bugfix in the PR description.

Done!

**Pull request merged**